dev_guide/cron_jobs.adoc

```diff
@@ -69,25 +49,64 @@ kind: CronJob
 metadata:
   name: pi
 spec:
-  schedule: "*/1 * * * *"  <1>
-  jobTemplate:             <2>
+  schedule: "*/1 * * * *"       <1>
+  concurrencyPolicy: "Replace"  <2>
+  startingDeadlineSeconds: 200  <3>
+  suspend: false                <4>
+  successfulJobsHistoryLimit: 3 <5>
+  failedJobsHistoryLimit: 1     <6>
+  jobTemplate:                  <7>
     spec:
       template:
         metadata:
-          labels:          <3>
+          labels:               <8>
             parent: "cronjobpi"
         spec:
           containers:
           - name: pi
             image: perl
             command: ["perl",  "-Mbignum=bpi", "-wle", "print bpi(2000)"]
-          restartPolicy: OnFailure <4>
+          restartPolicy: OnFailure <9>
 ----
 
-1. Schedule for the job. In this example, the job will run every minute.
-2. Job template. This is similar to the xref:../dev_guide/jobs.adoc#creating-a-job[job example].
-3. Sets a label for jobs spawned by this cron job.
-4. The restart policy of the pod. This does not apply to the job controller. See xref:../dev_guide/jobs.adoc#creating-a-job-known-issues[Known Issues and Limitations] for details.
+<1> The schedule field for the job is specified in 
+    link:https://en.wikipedia.org/wiki/Cron[cron format]. In this example, the
+    job runs every minute.
+<2> The concurrency policy specifies how to treat concurrent jobs
+    within a cron job. Only one of the following policies can be specified. By
```

Comment:
I'd make this sentence active. I'd also state that `Allow` is the default value in its definition so that the intro to the list flows smoothly into the list itself.